lib/ansible/modules/net_tools/hetzner_failover_ip_info.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019 Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: hetzner_failover_ip_info
+version_added: "2.9"
+short_description: Retrieve information on Hetzner's failover IPs
+author:
+  - Felix Fontein (@felixfontein)
+description:
+  - Retrieve information on Hetzner's failover IPs.
+seealso:
+  - name: Failover IP documentation
+    description: Hetzner's documentation on failover IPs.
+    link: https://wiki.hetzner.de/index.php/Failover/en
+  - module: hetzner_failover_ip
+    description: Manage failover IPs.
+extends_documentation_fragment:
+  - hetzner
+options:
+  failover_ip:
+    description: The failover IP address.
+    type: str
+    required: yes
+'''
+
+EXAMPLES = r'''
+- name: Get value of failover IP 1.2.3.4
+  hetzner_failover_ip_info:
+    hetzner_user: foo
+    hetzner_pass: bar
```

Comment:
Yeah, I found someting ;)

~~~diff
-    hetzner_pass: bar
+    hetzner_password: bar
~~~

Reply:
Ooops, I totally forgot about that example... I've now fixed it. Thanks for spotting this! :)